docs/Installation.md

```diff
@@ -214,6 +220,56 @@ service apache24 restart
 Use the procedure for installation on [Debian](#2-debian).
 
 
+## Serving the GUI and API from a custom base url
+
+In some cases you may want to customize the application to change the base url
+from wich the GUI is served.
+
+To achieve that you will either need to change the `<base>` in the `index.html`
+or build the application yourself.
+
+### Change the base url in index.html
+
+Locate the line `<base href="/">` and replace the `href` value with the path you
+want to server the GUI from, e.g. `<base href="/zonemaster/">`. Don't forget the
+trailing `/`.
+
+**NOTE:** Don't forget to update the base after each upgrade as the file will be
+overwritten.
```

Comment:
That is also true for the web server configuration. And it is true that the custom build has to be done at every update.

Reply:
I have rephrased this